tests/e2e/privacy.spec.js

```diff
@@ -56,50 +63,143 @@ describe('Privacy', () => {
     });
   });
 
-  describe('Ad-Blocking', () => {
+  describe.only('Ad-Blocking', () => {
     beforeEach(() => updatePrivacySettings('ad-blocking', false));
 
-    const WEBSITE_URL = 'https://www.onet.pl/';
-    const SELECTOR = 'div[class^="AdSlotPlaceholder_"]';
-
-    it('displays ads on a page', async () => {
-      await browser.url(WEBSITE_URL);
+    const SELECTOR = 'ad-slot';
 
-      let displayed = false;
-      for (const ad of await $$(SELECTOR)) {
-        if (displayed) break;
-        displayed = await ad.isDisplayed();
-      }
-
-      expect(displayed).toBe(true);
+    it('does not block ads on a page', async () => {
+      await browser.url(PAGE_URL);
+      await expect($(SELECTOR)).toBeDisplayed();
     });
 
     it('blocks ads on a page', async () => {
       await updatePrivacySettings('ad-blocking', true);
 
+      await browser.url(PAGE_URL);
+      await expect($(SELECTOR)).not.toBeDisplayed();
+    });
+  });
+
+  describe('Anti-Tracking', () => {
```

Comment:
only noticed now, but if this is mocha then please use function statements instead of arrow functions

Reply:
Why?

Reply:
https://mochajs.org/#arrow-functions

Reply:
Today we use wdio, tomorrow it may be using puppeteer, but most likely we will always use mocha :)

Reply:
You haven't convinced me, but I have changed it, as this a detail - if you think that's better, let it be ;)